README.md

```diff
@@ -10,6 +10,10 @@ use the Checkstyle plugin and the [config file](config/checkstyle.xml).
 
 To run checkstyle, execute `mvn clean verify -P checkstyle`
 
+## Running
+To run application with docker compose you need to create backend network first, we already have the `docker-compose-network.yaml`
+so just run the command `docker compose -f docker-compose.yml -f docker-compose-network.yaml up`
```

Comment:
Did you mean docker-compose-network.yml ?

Reply:
Fixed